cms/static/js/views/list.js

```diff
@@ -11,6 +11,8 @@
  *   of items this list contains.  For example, 'Group Configuration'.
  *   Note that it must be translated.
  * - emptyMessage (string): Text to render when the list is empty.
+ * - restrictEditing (bool) : Boolean flag for whether or not to restrict and hide editing options.
+ * - scheme (string): Scheme name for content group scheme
```

Comment:
js/views/list is a generic list view. It shouldn't have the notion of a "scheme".